### PR TITLE
Update katib example to use user profile

### PR DESCRIPTION
**Pull request by richardsliu**
https://github.com/kubeflow/website/issues/1599

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1633)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1633--competent-brattain-de2d6d.netlify.com/docs/components/hyperparameter-tuning/hyperparameter/#example-using-random-algorithm

File: content/docs/components/hyperparameter-tuning/hyperparameter.md [hunk 1]
```diff
@@ -109,7 +109,17 @@ Run the following command to launch an experiment using the random algorithm
 example:
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha3/random-example.yaml
+curl https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha3/random-example.yaml --output random-example.yaml
```
Comment (sarahmaddox) [inline content/docs/components/hyperparameter-tuning/hyperparameter.md]: This command doesn't actually launch the experiment, but the introductory sentence implies that it does.

A quick fix is to change "command" to "commands" on line 108.

The longer fix, which perhaps results in a clearer doc, is to change line 108 to this: "Run the following commands to launch an experiment using the random algorithm example:" and then make the subsequent points into a numbered list.

Comment (richardsliu) [reply content/docs/components/hyperparameter-tuning/hyperparameter.md]: Done, PTAL

Comment (sarahmaddox) [reply content/docs/components/hyperparameter-tuning/hyperparameter.md]: Looks great!

**Comment by jal06** [comment]
Thanks @richardsliu - just one suggestion in the kubectl describe experiment commands, please coud you replace the namespace kubeflow by the user profile namespace ?
The command `kubectl -n kubeflow describe experiment random-example` returns: 
`Error from server (NotFound): experiments.kubeflow.org "random-example" not found`

Instead of `kubectl -n kubeflow describe experiment random-example` , I think that `kubectl -n <user profile namespace> describe experiment random-example` should be better.

This replacement must be done for the 3 examples 

- random-example
- tfjob-example
- pytorchjob-example

**Pull request merged**